package.json

```diff
@@ -38,8 +39,10 @@
   },
   "scripts": {
     "test": "./bin/_lab -fL -t 100 -m 3000",
+    "posttest": "npm run lint-md",
     "test-cov-html": "./bin/_lab -fL -r html -m 3000 -o coverage.html",
-    "lint": "./bin/_lab -d -f -L"
+    "lint": "./bin/_lab -d -f -L",
+    "lint-md": "eslint --config hapi --rule 'strict: 0, eol-last: 0' --plugin markdown --ext md  ."
```

Comment:
In https://github.com/hapijs/hapijs.com/issues/217 we talked about what rules to allow/disallow, and we agreed on turning off hapi-scope-start.